ext/node/polyfills/testing.ts

```
@@ -123,6 +127,10 @@ class NodeTestContext {
             if (!newNodeTextContext[skippedSymbol]) {
               throw err;
             }
+          } finally {
+            try {
+              await after();
+            } catch { /* pass */ }
```

Does Node actually ignore error from `after` hooks?

---

Only when test is already failing. Fixed.

---

Got it